test/saithrift/tests/sail3.py

```diff
@@ -48,6 +48,9 @@ def runTest(self):
         nhop1 = sai_thrift_create_nhop(self.client, addr_family, ip_addr1, rif_id1)
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask1, rif_id1)
 
+        # Waiting the end of configuration ...
+        wait_till_configuration_will_end()
```

Comment:
We removed wait_till_configuration_will_end() function. On HW all works fine without delay.